### ISSUE TITLE
[Bug]  Latest docker image version should be 2.11.0

### ISSUE THREAD

**Issue by nodece** [apache/pulsar#19544]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### Minimal reproduce step

```
docker run -it -p 6650:6650 -p 8080:8080  apachepulsar/pulsar:latest bin/pulsar standalone
```

Log version:
```
2023-02-16T15:22:43,429+0000 [main] INFO  org.apache.pulsar.broker.PulsarService - Starting Pulsar Broker service; version: '2.10.1'
```

### What did you expect to see?

The latest image version is Pulsar@2.10.1, the digest is 4a952b3c662b, this should be an error version.

The Pulsar 2.11.0 has been released, so the latest image should be 2.11.0, not 2.10.1.

Please let me know if I miss anything.

### What did you see instead?

-

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by tisonkun** [comment]
cc @shibd @codelipenghui @Technoboy-

**Comment by shy-share** [comment]
@nodece You can specify the version
```bash
docker run -it -p 6650:6650 -p 8080:8080 --mount source=pulsardata,target=/pulsar/data --mount source=pulsarconf,target=/pulsar/conf apachepulsar/pulsar:2.11.0 bin/pulsar standalone
```
@tisonkun might just be the latest question. It is possible to use version 2.11.0

**Comment by nodece** [comment]
I know I can specify the Pulsar version. I just want to point this issue.

I guess the 2.10.1 has been released after 2.11.0, so 2.10.1 is the mater version. Maybe we should notice this issue.

**Comment by nodece** [comment]
This issue has been resloved.

**Comment by michaeljmarshall** [comment]
> I guess the 2.10.1 has been released after 2.11.0, so 2.10.1 is the master version. Maybe we should notice this issue.

In my opinion, latest means the greatest semantic version, so 2.11.0 is greater than all 2.10.x releases.

In general, it's not recommended to use the `latest` tag, but I do think we should only change the `latest` tag when we have a new greatest version.
